IngresseSDK/Model/Owner.swift

```diff
@@ -0,0 +1,35 @@
+//
```

Comment:
Owner and Operator classes are exactly equal, turn them into one and use for both, like User (maybe some class named User with these exact attributes already exists in the SDK)